src/main/resources/db/changelog/changelog_test.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog
```

Comment:
why do you need separate changelog for tests? is should use you already have

Reply:
at least i dont wanna ceate quarz tables

Reply:
test should run changelog you already have